Baby Monitor/Source Files/AppDependencies.swift

```diff
@@ -51,7 +51,6 @@ final class AppDependencies {
     private(set) lazy var webRtcClient: () -> WebRtcClientManagerProtocol = {
         WebRtcClientManager(
             peerConnectionFactory: self.peerConnectionFactory,
```

Comment:
Why have you decided to not to inject connection checker?

Reply:
Not needed when devices are already connected via WebSocket and streaming. `ConnectionChecker` chacks only Bonjour-type connection